test/chplcheck/UnusedTaskIntent.good-fixit

```diff
@@ -0,0 +1,32 @@
+module UnusedTaskIntent {
+  var A: [1..10] int = 10;
+  var B: int = 1;
+
+  forall 1..10 with (var x: int) {}
+
+  forall 1..10 with (var x = B) {}
+
+  @chplcheck.ignore("UnusedTaskIntent")
+  foreach 1..10
+    with (ref A,
+          in B) {}
+
+  coforall 1..10  {}
+
+  begin with (ref A,
+  + reduce B) {}
+
+  begin  {}
+
+  cobegin  { }
```

Comment:
There are double spaces here. It's not a huge deal but I thought I'd point it out.

Reply:
I think I can actually fix that